src/tls_openssl.c

```diff
@@ -38,100 +37,100 @@ static bool add_ca_certs(SSL_CTX *ctx, STACK_OF(X509_INFO) * certs) {
   return true;
 }
 
-static EVP_PKEY *load_key(const char *key, int key_len) {
-  BIO *key_bio = BIO_new_mem_buf(key, key_len);
-  if (!key_bio) return NULL;
```

Comment:
Is this safe ? No longer checking for NULL before calling (nor freeing later)

Reply:
ditto